website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,215 @@
+!!!
+
+## Interaction
+
+The Rich Tooltip supports two different interaction methods: a `hover/focus` ("soft") interaction and a `click` interaction. Which one to use depends on the type and complexity of content displayed by the component, whether _temporarily persisting_ the content in the UI is necessary or beneficial to the user, and whether the element toggling the Rich Tooltip is interactive or not.
+
+1. `Hover/focus` (default): displays when the mouse enters the toggle or the toggle receives focus. The `RichTooltip` has a timeout by default and will persist for 500 milliseconds (0.5 seconds) _after_ the mouse leaves the toggle, or the focus is removed from it.
+2. `On click`: displays when the user clicks the toggle with a mouse or if the toggle receives a keyboard event (`spacebar`, `enter/return`). If the content of the toggle element is interactive, an `on click` interaction _cannot_ be used.
+
+Independent of which interaction is used, the Rich Tooltip can be dismissed by clicking outside of the component or with the `ESC` key.
+
+## Toggle
+
+The Rich Tooltip supports both a default toggle (InfoText) to use the component in a consistent manner, but also supports generic or custom elements passed to the toggle element.
+
+### InfoText
+
+Be default, we provide the [InfoText](/components/rich-tooltip?tab=specifications#infotext-1) component which accounts for the majority of use-cases for the Rich Tooltip. This component ensures that the toggle is perceivable, visually consistent, and can be used inline with other content or standalone as part of the layout flow.
+
+!!! Info
+
+For now, the InfoText component should only be used with the Rich Tooltip. In Figma, this component is published separately for flexibility, while in Ember the component is coupled with the Rich Tooltip.
+!!!
+
+#### How to use
+
+Consider these guidelines when using the InfoText component:
+
+- Use the text + trailing icon variant by default.
+- Use the `isUnderlineOnly` variant within text layers and blocks of text (only applicable to Figma).
+- When using the `iconOnly` variant without text, ensure that the component is paired with other textual elements (like a headline) or elements being described by the Rich Tooltip. Don’t use it in isolation or on it’s own.
+- When used in a block of text, we recommend omitting the icon as this can unnecessarily break up the reading flow of the content.
+
+#### Size
+
+![Sizes of the InfoText](/assets/components/rich-tooltip/rich-tooltip-info-text-sizes.png)
+
+#### Icon
+
+![Sizes of the InfoText](/assets/components/rich-tooltip/rich-tooltip-info-text-icon.png)
+
+#### Underline
+
+Only available in the InfoText component in Figma, the `isUnderlineOnly` property allows the component to be used **inline** a block of text like a paragraph.
+
+![Sizes of the InfoText](/assets/components/rich-tooltip/rich-tooltip-info-text-underline.png)
```

Comment:
Alt text needs to be updated for this image. This screenshot gets lost in the rest of the documentation. Can you include more of the UI so this looks like an example instead of continued documentation?

Reply:
Cleaned up a bit and added some more contextual UI.